be/src/formats/parquet/column_reader.cpp

```diff
@@ -246,18 +249,73 @@ class MapColumnReader : public ColumnReader {
     std::unique_ptr<ColumnReader> _value_reader;
 };
 
+class StructColumnReader : public ColumnReader {
+public:
+    explicit StructColumnReader(const ColumnReaderOptions& opts) {}
+    ~StructColumnReader() override = default;
+
+    Status init(const ParquetField* field, std::vector<std::unique_ptr<ColumnReader>> child_readers) {
```

Comment:
why not use a rvalue reference here because you call this function with 'std::move'  parameter?

Reply:
Good idea, changed.